keps/sig-node/2400-node-swap/README.md

```diff
@@ -401,8 +401,12 @@ For alpha:
   and further development efforts.
   - Focus should be on supported user stories as listed above.
 
-Once this data is available, additional test plans should be added for the next
-phase of graduation.
+For beta:
```

Comment:
I think we need to provide clear guidance on how to configure eviction-hard and eviction-soft.

It is not clear to me the behavior we will see for how the kubelet reports memory pressure for scenario 1 and 2 in this kep.  I will try and work that backward as well.

Reply:
Pushed 8277301aae07f6bab8208ae495a63078af7e1057 to address

Reply:
eviction manager controls if a node is under memory pressure.  it runs a check using `AvailableBytes` and `WorkingSetBytes`, neither appears to track swap which appears to only be tracked in `RssBytes`.  for scenario 1, when node has swap enabled, but workloads cannot consume it, I _think_ the worst outcome is we evict unnecessarily because swap would be available.  For scenario 2, its less clear to me and we need to test the behaviors we desire.